Dockerfile

```diff
@@ -67,15 +67,19 @@ ENV HOME /home/$PROJECTOR_USER_NAME
 ENV PROJECTOR_CONFIG_DIR $HOME/.config
 RUN set -ex \
     && microdnf install -y --nodocs \
-    shadow-utils wget git nss procps findutils which socat \
-    # Packages required by JetBrains products.
-    libsecret jq \
+    shadow-utils wget git nss procps findutils which socat jq \
     # Java 11 support
     java-11-openjdk-devel \
     # Python support
     python2 python39 \
     # Packages needed for AWT.
     libXext libXrender libXtst libXi libX11-xcb mesa-libgbm libdrm freetype \
+    # Install libsecret for the particular platform
+    && { [ $(uname -m) == "s390x" ] && yum install -y --nodocs \
+        https://rpmfind.net/linux/fedora-secondary/releases/34/Everything/s390x/os/Packages/l/libsecret-0.20.4-2.fc34.s390x.rpm \
+        https://rpmfind.net/linux/fedora-secondary/development/rawhide/Everything/s390x/os/Packages/l/libsecret-devel-0.20.4-3.fc35.s390x.rpm || true; } \
+    && { [ $(uname -m) == "ppc64le" ] && yum install -y --nodocs libsecret https://rpmfind.net/linux/centos/8-stream/BaseOS/ppc64le/os/Packages/libsecret-devel-0.18.6-1.el8.ppc64le.rpm || true; } \
+    && { [ $(uname -m) == "x86_64" ] && yum install -y --nodocs libsecret || true; } \
```

Comment:
I think the || true can be dangerous as it can silently report success even if there is a failure

Reply:
True... but this is the same code as in theia: https://github.com/eclipse-che/che-theia/pull/1167/files Perhaps you have a better way to do "install if not installed and don't fail if already installed; but if could not install, fail build" ?

Reply:
> have a better way to do "install if not installed and don't fail if already installed; but if could not install, fail build" ?
I don't know how it's possible as either it's installed and then we should not install
or if it's there we should not install

but we can check if package is there using `rpm` command and then depending on the result we can proceed to install or say 'already installed'

want to know if bash is there
```bash
$ rpm -q bash
```

```
$ if rpm --quiet -q bash; then echo "bash is installed"; else echo "bash is not installed"; fi
```

Reply:
From what I see, `rpm` is bundled into `registry.access.redhat.com/ubi8-minimal:8.4-205`. So, either libsecret rpms can be installed through providing them as local files or installed through `rpm -ih http://...`.